Cargo.toml

```diff
@@ -34,10 +27,10 @@ package_aliases_dir = "bazel/cargo"
 workspace_path = "//bazel/cargo"
 genmode = "Remote"
 
-[[example]]
-name = "hello_world"
-path = "examples/hello_world.rs"
-crate-type = ["cdylib"]
+# [[example]]
```

Comment:
Unlikely this can stay due to example's `Cargo.toml` now having dependency definitions. Commented out first for clarity but should just be removed.